.travis.yml

```diff
@@ -31,6 +33,7 @@ install:
  - cd libxsmm-master && make generator && sudo cp bin/libxsmm_gemm_generator /usr/bin && cd ..
 
 script:
+ - sphinx-build -n -b dummy Documentation Documentation/_build/html
```

Comment:
Is there a reason why you don't use the makefile in the Documentation folder?
(not sure how it's actually build by readthedocs)

Reply:
Yes, "make" would be better.